Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM ruby:2.3-alpine
+FROM ruby:2.4-alpine
 
 WORKDIR /usr/src/app
-COPY Gemfile /usr/src/app/
-COPY Gemfile.lock /usr/src/app/
 
-RUN gem install bundler && \
-    bundle install -j 4 && \
-    rm -fr /usr/share/ri
+RUN adduser -u 9000 -D app
 
-RUN apk --update add git
+COPY Gemfile Gemfile.lock /usr/src/app/
+
+RUN gem install bundler && \
+    bundle install -j 4 --without=test && \
```

Comment:
I appreciate that you always look for ways to make our docker images smaller, but I think this is going further than necessary: running `bundle install` every time you run tests locally makes the local feedback cycle much too slow. The test gems really aren't that big, we install them into almost all of our production images, so I'd prefer we keep these.

Reply:
OK. I'll take a look a bit later at this.